src/app/clusters/operational-credentials-server/operational-credentials-server.cpp

```diff
@@ -565,6 +567,17 @@ bool emberAfOperationalCredentialsClusterAddNOCCallback(app::CommandHandler * co
     // Notify the secure session of the new fabric.
     commandObj->GetExchangeContext()->GetSessionHandle()->AsSecureSession()->NewFabric(fabricIndex);
 
+    // Set the Identity Protection Key (IPK)
+    VerifyOrExit(nullptr != groups, nocResponse = ConvertToNOCResponseStatus(CHIP_ERROR_INTERNAL));
+    VerifyOrExit(ipkValue.size() != Crypto::CHIP_CRYPTO_SYMMETRIC_KEY_LENGTH_BYTES,
+                 nocResponse = ConvertToNOCResponseStatus(CHIP_ERROR_INTERNAL));
```

Comment:
This should yield an INVALID_COMMAND status. Does CHIP_ERROR_INTERNAL map to that?

Reply:
It does not.  But more to the point, we should not be doing the exit to where we send a NOCResponse here at all; we should in fact be sending an error status.